### PR TITLE
Update to integration tests in Landkid

### PR DESCRIPTION
**Pull request by RajaBellebon**
### Description

I started modifying the username and a couple of testing-related things then I discover we can't run integration tests due to the Bitbucket VPN limitation.

Maybe in a near future, we should
* move Landkid to BB
* move the integration test to the Landkid Atlassian Frontend monorepo

#### Main changes
* Comment all the integration section on Circle CI
* Upgrade Cypress to latest with new goodies and use Chrome (latest)
* Update username to mine as I am the owner of the BB testing repo
* Add some testing documentation to run tests locally on the dev instance

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,9 +19,9 @@
     "clean": "rimraf dist",
     "test:unit": "jest",
     "pretest:integration": "node tools/test-local.js",
-    "test:integration": "yarn test:integration:staging",
     "posttest:integration": "node tools/test-local.js --revert",
-    "test:integration:staging": "cypress run --reporter junit",
+    "test:integration": "cypress run --browser chrome --reporter junit",
```
Comment (RajaBellebon) [inline package.json:23]: I ma not sure we want to keep the `staging` as it doe snto make sense to me

File: tests/integration/utils/cypress-commands.ts [hunk 2]
```diff
@@ -30,52 +29,60 @@ Cypress.Commands.add('createLandRequest', (title: string, isSuccessful: boolean)
...
-      source: {
-        branch: {
-          name: title,
-        },
-      },
-      destination: {
-        branch: {
-          name: 'master',
-        },
+  try {
```
Comment (RajaBellebon) [inline tests/integration/utils/cypress-commands.ts:32]: wrapping into a try catch if not errors were swallowed

**Comment by RajaBellebon** [comment]
@luciaquirke do you want to test the `TESTING.md` and see if that makes sense and you are able to test locally after deploying to dev.

File: tests/integration/utils/cypress-commands.ts (hunk 2 above)
Comment (lukebatchelor) [reply tests/integration/utils/cypress-commands.ts:32]: The diff is super cooked here, was the only change moving this into the try catch?

**Review by lukebatchelor — APPROVED** [review]

**Comment by luciaquirke** [comment]
> @luciaquirke do you want to test the `TESTING.md` and see if that makes sense and you are able to test locally after deploying to dev.

Two nitpicks for the doc:
- should specify where in the publish step's page the docker image number is found
- should specify which file in the atlassian-frontend-landkid-deployment repository holds the LANDKID_VERSION var

**Comment by RajaBellebon** [comment]
> > @luciaquirke do you want to test the `TESTING.md` and see if that makes sense and you are able to test locally after deploying to dev.
> 
> Two nitpicks for the doc:
> 
> * should specify where in the publish step's page the docker image number is found
> * should specify which file in the atlassian-frontend-landkid-deployment repository holds the LANDKID_VERSION var

Second bullet it is already linked - maybe it did not work well.
I will update the first bullet

**Review by luciaquirke — APPROVED** [review]

**Pull request merged**